src/OrbitBase/TemporaryFile.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitBase/TemporaryFile.h"
+
+#include <stdlib.h>
+
+namespace orbit_base {
+
+void TemporaryFile::Create() noexcept {
+  std::error_code error;
+  std::filesystem::path temporary_dir = std::filesystem::temp_directory_path(error);
+  std::string file_path = (temporary_dir / "orbit_XXXXXX").string();
```

Comment:
So this is because we want to it start with "orbit_"? Is it necessary? Why not just use `std::tmpnam`?

Reply:
tmpnam is not safe it generates a lot of warnings. The reason is that it does not actually create a file.